packages/block-library/src/navigation-menu-item/save.js

```diff
@@ -1,12 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	InnerBlocks,
+} from '@wordpress/block-editor';
+
 export default function save( { attributes } ) {
 	return (
```

Comment:
nit: We could destructure the attributes here so that we're not repeating `attributes` below so often.

```js
const { destination, nofollow, title, opensInNewTab, label } = attributes;
```